clickhouse-client/src/test/java/com/clickhouse/client/ClickHouseServerForTest.java

```diff
@@ -157,8 +164,83 @@ public class ClickHouseServerForTest {
                     .withNetwork(network)
                     .withNetworkAliases("clickhouse")
                     .waitingFor(Wait.forHttp("/ping").forPort(ClickHouseProtocol.HTTP.getDefaultPort())
+                            .forStatusCode(200).withStartupTimeout(Duration.of(600, SECONDS)));
+        }
+
+        if (clickhouseTLSServer != null) { // use external server
```

Comment:
Why separate container? Can we reuse one container/external server which has both http & https ports exposed?